api/lib/infrastructure/repositories/organization-for-admin-repository.js

```diff
@@ -70,4 +70,28 @@ module.exports = {
 
     return _toDomain(organization);
   },
+
+  async save(organization) {
+    if (!organization.archiveDate || !organization.archivedBy) {
+      throw new MissingAttributesError();
+    }
+
+    if (organization.newInvitationStatus) {
```

Comment:
J'ai du mal à voir pourquoi newInvitationStatus pourrait ne pas être défini `if (organization.newInvitationStatus)`
De ce que je comprends, newInvitationStatus est créé dans la méthode archive du modèle OrganizationForAdmin
` this.newInvitationStatus = OrganizationInvitation.StatusType.CANCELLED;` 
et StatusType vient du modèle `api/lib/domain/models/OrganizationInvitation.js`

J'ai pas regardé les tests pour comprendre un peu mieux ceci dit.

Reply:
ce n'est plus d'actualite

Reply:
Cool !